dotCMS/src/main/java/com/dotmarketing/business/UserFactoryLiferayImpl.java

```diff
@@ -182,9 +182,40 @@ public User loadByUserByEmail(String email) throws DotDataException, DotSecurity
 			throw new DotDataException(e.getMessage(), e);
 		}
 	}
-	
+
+	@Override
+	public long getCountUsersByName(String filter) throws DotDataException {
+		filter = SQLUtil.sanitizeParameter(filter);
+		DotConnect dotConnect = new DotConnect();
+		boolean isFilteredByName = UtilMethods.isSet(filter);
+		filter = (isFilteredByName ? filter : "");
+		StringBuffer baseSql = new StringBuffer("select count(*) as count from user_ where companyid = ? and userid <> 'system' ");
+		String userFullName = dotConnect.concat( new String[]{ "firstname", "' '", "lastname" } );
+
+		if( isFilteredByName ) {
+			baseSql.append(" and lower(");
+			baseSql.append(userFullName);
+			baseSql.append(") like ?");
+		}
+
+		baseSql.append(" and delete_in_progress = ");
+		baseSql.append(DbConnectionFactory.getDBFalse());
+
+		String sql = baseSql.toString();
+		dotConnect.setSQL(sql);
+		Logger.debug( UserFactoryLiferayImpl.class,"::getCountUsersByName -> query: " + dotConnect.getSQL() );
+
+		dotConnect.addParam(PublicCompanyFactory.getDefaultCompanyId());
+		if(isFilteredByName) {
+			dotConnect.addParam("%"+filter.toLowerCase()+"%");
+		}
+
+		return dotConnect.getInt("count");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "count" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)